docs/spec/api.md

```diff
@@ -3651,9 +3651,9 @@ The following parameters should be specified on the request:
 
 ```
 202 Accepted
-Content-Length: 0
 Location: /v2/<name>/blobs/uploads/<uuid>
-Range: 0-0
+Range: bytes=0-<offset>
```

Comment:
Can you update this in the [source](https://github.com/distribution/distribution/blob/ad8f5caba068b2d4b83ba5a1c3e9d74e1da2c16e/registry/api/v2/descriptors.go#L1055), instead of directly in `api.md`? 

Same for the `Content-Length`, we need to move [`contentLengthZeroHeader`](https://github.com/distribution/distribution/blob/ad8f5caba068b2d4b83ba5a1c3e9d74e1da2c16e/registry/api/v2/descriptors.go#L1046) down.

We need to improve our docs and CI to detect these issues (https://github.com/distribution/distribution/issues/3433).